package.json

```diff
@@ -20,8 +20,8 @@
     "url": "http://www.opensource.org/licenses/MIT"
   },
   "dependencies": {
-    "jsonwebtoken": "^5.0",
-    "passport-http-bearer": "^1.0.1"
+    "jsonwebtoken": "7.4.1",
```

Comment:
Adding comment here because I cannot comment on lines that were not changed. Shouldn't passport-http-jwt-bearer version be updated to 0.1.4-ucep in package.json?